### ISSUE TITLE
Support for Samsung EHS Mono

### ISSUE THREAD

**Issue by TheSimpleZ** [veista/smartthings#85]
Hello! 
Could you please add full support for my heating system?  
Currently, it displays some values related to the water tank, but nothing related to the indoor heating system.

![image](https://github.com/veista/smartthings/assets/2884929/a8452a37-0161-4eff-8587-9c96486e1f2e)

I'd like to be able to set the temperature of the water tank and indoor heating system. As well as choosing the HVAC mode. 
Other great features would be to be able to set Away and Quiet mode.

These are all controllable through the Smartthings app. I'm happy to help in any way possible!

**Comment by ColinSainsbury** [comment]
I too have the AE120BXYDEG/EU and would love to see this enabled. I am hoping to find a way to detect export from my solar PV/Tesla Powerwall of greater than 1kW and turn on the DHW of the heat pump. I have a MyEnergi Eddi as a solar diverter but the ASHP is more efficient. 

@TheSimpleZ have you been able to get the Temperature Measurement to update? On my system it only updates when the integration starts and remains at the same value until the next time I restart the integration/HomeAssistant.

**Comment by veista** [comment]
See discussions

**Comment by TheSimpleZ** [comment]
@ColinSainsbury Yeah, temperature measurement works fine on mine. I haven't really done anything special to make it work. It simply has been working, both with the official Smartthings integration and this one.

**Comment by ColinSainsbury** [comment]
@TheSimpleZ Track it through the day, it never changes for me. By reloading the entity every 5 minutes I get near real time Hot Water temperature. You might ask why do I care? Well, my wife does not want the water to be cold when she goes for her shower and sometimes my daughter uses a lot!

**Comment by TheSimpleZ** [comment]
I don't have to do any restarts to get updates, and they do seem to happen.
![image](https://github.com/veista/smartthings/assets/2884929/46eb8ec5-d584-45ea-8819-9327bc85fc62)

However, it looks like the value is being updated quite seldom. Like every 3 hours or so. Next time we use a lot of hot water I'll check if the temperature in HA actually matches what's on the display.

**Comment by ColinSainsbury** [comment]
I eventually removed and reinstalled the integration and have success with updating the DHW temperature automatically.
